src/App/App.less

```diff
@@ -1,15 +1,9 @@
-.forside {
+.Side {
```

Comment:
`side` er kanskje litt generelt. F.eks. kan det være forvirrende om denne klassen gjelder for alle sider eller ikke.
Kan bruke liten forbokstav i CSS klasser